<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T3_AF_vocalText</class>
 <widget class="QDialog" name="T3_AF_vocalText">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="_exitPushBtn_">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset;color:white</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="_backgroundLabel_">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>405</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Pictures/faceHistory_background.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="_userTypeLabel_">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>用户类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="_voiceLabel_">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>文本编辑：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="_userTypeComboBox_">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:1;
border-style:dashed;
border-color:white;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="_voiceTextEdit_">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-width:1;
border-style:dashed;
color:white;
border-color:white;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_saveBtn_">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>345</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:
url(:/Pictures/face_button.png);color:white</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="_exampleText_">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>231</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string> 说明：

 选择用户类型，即可修改该类用户对应欢迎用语。
（姓名）：识别对象姓名。
（性别）：识别对象性别，将替换为“先生”/“女士”。

 备注：请使用中文标点符号。</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="_back2_">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>202</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/vocal_back2.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListView" name="_groups_">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>67</y>
     <width>175</width>
     <height>234</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset;color:white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_groupName_">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>315</y>
     <width>175</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/faceLog_textEdit.png);
color:white</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="_add_">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/faceLog_button.png);
color:white</string>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_delete_">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>350</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/faceLog_button.png);
color:white</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="_wholeBack_">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/confirmExit_background.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>_wholeBack_</zorder>
  <zorder>_backgroundLabel_</zorder>
  <zorder>_exitPushBtn_</zorder>
  <zorder>_userTypeLabel_</zorder>
  <zorder>_voiceLabel_</zorder>
  <zorder>_userTypeComboBox_</zorder>
  <zorder>_voiceTextEdit_</zorder>
  <zorder>_saveBtn_</zorder>
  <zorder>_exampleText_</zorder>
  <zorder>_back2_</zorder>
  <zorder>_groups_</zorder>
  <zorder>_groupName_</zorder>
  <zorder>_add_</zorder>
  <zorder>_delete_</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
